package.json

```diff
@@ -399,19 +422,25 @@
                 "scopeName": "markdown.math_display",
                 "path": "./syntaxes/math_display.markdown.tmLanguage.json",
                 "injectTo": [
-                    "text.html.markdown"
+                    "text.html.markdown",
+                    "text.html.markdown.jsx"
                 ]
             },
             {
                 "scopeName": "markdown.math_inline",
                 "path": "./syntaxes/math_inline.markdown.tmLanguage.json",
                 "injectTo": [
-                    "text.html.markdown"
+                    "text.html.markdown",
+                    "text.html.markdown.jsx"
                 ]
             },
             {
                 "scopeName": "text.katex",
-                "path": "./syntaxes/katex.tmLanguage.json"
+                "path": "./syntaxes/katex.tmLanguage.json",
+                "injectTo": [
+                    "text.html.markdown",
+                    "text.html.markdown.jsx"
+                ]
```

Comment:
I cannot understand this change.

---

```json
{
    "scopeName": "text.katex",
    "path": "./syntaxes/katex.tmLanguage.json"
}
```

was intended to simply import the TM grammar, because `math_display.markdown.tmLanguage.json` and `math_inline.markdown.tmLanguage.json` reference `text.katex` scope.

I think there should be no need to inject it into any scope.